cves/CVE-2017-15706.yml

```diff
@@ -132,8 +144,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: The documentation of common gateway interfaces
+  name: docs
```

Comment:
I would recommend going with "Documentation" just to be more clearer and consistent.

Reply:
resolved.